data-access-services/docker-build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# Build and verify all modules
+mvn clean package
+
+# Build containers for individual services
+mvn -f ./transactions-service/pom.xml \
+ spring-boot:build-image \
+ -Dspring-boot.build-image.imageName=core-banking/transactions-service \
+ -DskipTests
+mvn -f ./accounts-service/pom.xml \
+ spring-boot:build-image \
+ -Dspring-boot.build-image.imageName=core-banking/accounts-service \
+ -DskipTests
```

Comment:
We would have to run both the sh scripts and then the docker compose now to set up system right?